applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_implicit_dynamic_solver.py

```diff
@@ -64,8 +64,8 @@ def InitializeSolutionStep(self):
         super(ImplicitMechanicalSolver, self).InitializeSolutionStep()
 
         # Some pre-processes may affect the system of equations, we rebuild the equation ids
-        proces_info = self.main_model_part.ProcessInfo
-        if proces_info[KratosMultiphysics.STEP] == 1 and proces_info[StructuralMechanicsApplication.RESET_EQUATION_IDS]:
+        process_info = self.main_model_part.ProcessInfo
+        if process_info[KratosMultiphysics.STEP] == 1 and process_info[StructuralMechanicsApplication.RESET_EQUATION_IDS]:
```

Comment:
:-o what is this RESET_EQUATION_IDS  ... i see it existed before but it looks very scary to me.